backend/src/app/rpc/commands/binfile.clj

```diff
@@ -705,6 +736,11 @@
                   :expected-id expected-file-id
                   :hint "the penpot file seems corrupt, found unexpected uuid (file-id)"))
 
+      (when (contains? file :thumbnails)
+        (l/dbg :hint "updating thumbnails" ::l/sync? true)
+        (vswap! *state* update :thumbnails into (update-file-thumbnails file file-id'))
+        (l/dbg :hint "updated thumbnails" :thumbnails (:thumbnails @*state*) ::l/sync? true))
```

Comment:
this log entry can be very verbose when a file has a huge amount of thumbnails, consider using a single log entry and maybe using a counter of thumbnails instead of two